src/KafkaFlow.Abstractions/Acks.cs

```diff
@@ -1,9 +1,15 @@
 namespace KafkaFlow
 {
+    /// <summary>Acks enum values</summary>
     public enum Acks
     {
+        /// <summary>Leader</summary>
         Leader,
+        
+        /// <summary>All</summary>
         All,
+        
+        /// <summary>None</summary>
```

Comment:
In WithAcks method there is a description of each one, you can remove from there and put it here

Reply:
DONE